pytorch/training/docker/1.13/py3/example/Dockerfile.gpu.example.os_scan_allowlist.json

```diff
@@ -1,57 +1,90 @@
+        "name": "cryptography",
+        "package_manager": "PYTHONPKG",
+        "version": "38.0.4",
+        "release": null
+      },
+      "remediation": {
+        "recommendation": {
+          "text": "None Provided"
+        }
+      },
+      "cvss_v3_score": 7.5,
+      "cvss_v30_score": 0,
+      "cvss_v31_score": 7.5,
+      "cvss_v2_score": 0,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-3996.html",
+      "source": "UBUNTU_CVE",
+      "severity": "LOW",
+      "status": "ACTIVE",
+      "title": "CVE-2022-3996 - cryptography"
+    }
+  ],
+  "torch": [
+    {
+      "description": "[torch](https://pypi.org/project/torch) is a Tensors and Dynamic neural networks in Python with strong GPU acceleration\n\nAffected versions of this package are vulnerable to Command Injection in `torch.jit.annotations.parse_type_line` which can cause arbitrary code execution because `eval` is used unsafely.",
+      "vulnerability_id": "SNYK-PYTHON-TORCH-3149871",
+      "name": "SNYK-PYTHON-TORCH-3149871",
+      "package_name": "torch",
+      "package_details": {
+        "file_path": "opt/conda/lib/python3.9/site-packages/torch-1.13.0+cu117.dist-info/METADATA",
+        "name": "torch",
+        "package_manager": "PYTHONPKG",
+        "version": "1.13.0+cu117",
+        "release": null
+      },
+      "remediation": {
+        "recommendation": {
+          "text": "None Provided"
+        }
+      },
+      "cvss_v3_score": 9.8,
+      "cvss_v30_score": 0,
+      "cvss_v31_score": 9.8,
+      "cvss_v2_score": 0,
+      "cvss_v3_severity": "CRITICAL",
+      "source_url": "https://snyk.io/vuln/SNYK-PYTHON-TORCH-3149871",
+      "source": "SNYK",
+      "severity": "CRITICAL",
+      "status": "ACTIVE",
+      "title": "IN1-PYTHON-TORCH-3149871 - torch"
+    },
+    {
+      "description": " In PyTorch before trunk/89695, torch.jit.annotations.parse_type_line can cause arbitrary code execution because eval is used unsafely.",
+      "vulnerability_id": "CVE-2022-45907",
+      "name": "CVE-2022-45907",
+      "package_name": "torch",
+      "package_details": {
+        "file_path": "opt/conda/lib/python3.9/site-packages/torch-1.13.0+cu117.dist-info/METADATA",
+        "name": "torch",
+        "package_manager": "PYTHONPKG",
+        "version": "1.13.0+cu117",
+        "release": null
+      },
+      "remediation": {
+        "recommendation": {
+          "text": "None Provided"
+        }
+      },
+      "cvss_v3_score": 9.8,
+      "cvss_v30_score": 0,
+      "cvss_v31_score": 9.8,
+      "cvss_v2_score": 0,
+      "cvss_v3_severity": "CRITICAL",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-45907.html",
+      "source": "UBUNTU_CVE",
+      "severity": "MEDIUM",
+      "status": "ACTIVE",
+      "title": "CVE-2022-45907 - torch"
+    }
+  ]
 }
```

Comment:
Mind adding a new line under here for sanity? Notice the red circle on the end of the file?